compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/statements/BLangDo.java

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.ballerinalang.compiler.tree.statements;
+
+import org.ballerinalang.model.clauses.OnFailClauseNode;
+import org.ballerinalang.model.tree.NodeKind;
+import org.ballerinalang.model.tree.statements.BlockStatementNode;
+import org.ballerinalang.model.tree.statements.DoNode;
+import org.wso2.ballerinalang.compiler.tree.BLangNodeVisitor;
+import org.wso2.ballerinalang.compiler.tree.clauses.BLangOnFailClause;
+
+/**
+ * @since Swan Lake
+ */
```

Comment:
Lets give a brief description to this class

Reply:
Added in 3cd62b5aa3dd887f5272949bd3cbb00faad8030c